sdk/template/azure-template2/inc/azure/template2/version.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/template2/dll_import_export.hpp"
+
+#include <string>
+
+#define AZURE_TEMPLATE_VERSION_MAJOR 1
+#define AZURE_TEMPLATE_VERSION_MINOR 0
+#define AZURE_TEMPLATE_VERSION_PATCH 0
+#define AZURE_TEMPLATE_VERSION_PRERELEASE "beta.29"
+
+namespace Azure { namespace Template { namespace Details {
```

Comment:
template 2?

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1651